src/pages/home/report/ContextMenu/genericReportActionContextMenuPropTypes.js

```diff
@@ -1,12 +1,14 @@
 import PropTypes from 'prop-types';
-import reportActionPropTypes from '../reportActionPropTypes';
 
 const propTypes = {
     /** The ID of the report this report action is attached to. */
     reportID: PropTypes.string.isRequired,
 
-    /** The report action this context menu is attached to. */
-    reportAction: PropTypes.shape(reportActionPropTypes).isRequired,
+    /** The ID of report action this context menu is attached to. */
+    reportActionID: PropTypes.string.isRequired,
+
+    /** The ID of the current report of this report action is attached to. */
+    originalReportID: PropTypes.string.isRequired,
```

Comment:
Just need a bit of clarity on this one. Can you explain (maybe with an example) what you mean when you say in your proposal "The thread first chat originalReportID and reportID is different."?

Reply:
Ah ignore me, I see that we're using the same naming elsewhere!